frontend/model/state.js

```diff
@@ -348,6 +348,7 @@ const getters = {
     return Colors[state.theme]
   },
   isDarkTheme (state) {
+    if (!state.theme) state.theme = 'light' // TODO: check color available, prevent setting something differrent than available colors
```

Comment:
Do you plan to do this in this PR? If not, open an issue to not be forgotten or update the current one.